package.json

```
@@ -22,15 +22,15 @@
     "prebuild": "rm -fr dist/* && npm run gen-data-json",
     "version": "node scripts/set-version.js",
     "build": "tsc && node build.mjs",
-    "build:image": "npm run build && docker buildx build --tag pepr:dev .",
+    "build:image": "npm run build && docker buildx build --output type=docker --tag pepr:dev .",
```

```suggestion
    "build:image": "npm run build && docker buildx build --output type=docker --tag pepr:dev .",
```

I thought I only needed this, did you intend on adding this?

---

Yep, was intended -- it doesn't hurt the Linux box I work off of & it _definitely_ helps your Mac work so I don't see why we wouldn't just leave it in there. :man_shrugging: